script.js

```diff
@@ -0,0 +1,855 @@
+import Producto from "./Producto.js" 
+import Carrito from "./Carrito.js" 
+
+//DOM PARA EL PAGO_REALIZADO.HTML
+// funcion para modificar la informacion del pago
+// recupero los valores del sessionStorage
+let mdpPagoRealizado = sessionStorage.getItem('mdp')
+const emailPagoRealizado = sessionStorage.getItem('email')
+const totalPagoRealizado = sessionStorage.getItem('totalPago')
+
+// recupero los elementos desde el HTML donde se van a agregar los datos del session storage
+const emailElemento = document.getElementById('email-pago-realizado')
+const mdpElemento = document.getElementById('mdp-pago-realizado')
+const totalElemento = document.getElementById('total-pago-realizado')
+
+// agrego un evento cuando cargue el dom
+document.addEventListener('DOMContentLoaded', () => {
+    // si ninguno de los elementos es null recien ejecuta esto.
+    if(emailElemento && mdpElemento && totalElemento){
+        //asigno los datos
+        emailElemento.textContent = emailPagoRealizado
+
+        if(mdpPagoRealizado === "1"){
+            mdpPagoRealizado = "Debito"
+        }else if(mdpPagoRealizado === "2"){
+            mdpPagoRealizado = "Credito"
+        }
+        mdpElemento.textContent = mdpPagoRealizado
+
+        totalElemento.textContent = totalPagoRealizado
+    }
+    
+})
+
+// MANEJO DE LA PESTAÑANA PANTALLA_CARGA_COMPRA.HTML
+// funcion asincrona para obtener las frases de una api de mockAPI que cree yo.
+async function cargarApiFrases() {
+    fetch("https://67474f1a38c8741641d64755.mockapi.io/frases")
```

Comment:
Implementa el control de errores utilizando try-catch para manejar posibles problemas al cargar datos de la API.
```suggestion
       try {
        const response = await fetch("https://67474f1a38c8741641d64755.mockapi.io/frases");
        if (!response.ok) throw new Error('Error en la carga: ' + response.status);
        const data = await response.json();
        const frases = data.map(item => item.frase);
        crearFraseDeCompra(frases);
    } catch (error) {
        console.error("Error al cargar las frases:", error);
   }

```